### PR TITLE
feat(locale): Fully Improve zh_TW

### PR DESCRIPTION
**Pull request by CoolPlayLin**
# Works
- Add color module
- Add date module
- ~~Fix according to ISO codes~~

### CODE REVIEW

**Review by ST-DDT — DISMISSED** [review]

**Comment by ST-DDT** [comment]
Please run `pnpm run preflight`.

**Comment by CoolPlayLin** [comment]
@ST-DDT It appears this isn't my issue. It was caused in Comment

**Review by ST-DDT — APPROVED** [review]

File: src/locales/zh_TW/location/default_country.ts [hunk 1]
```diff
@@ -1 +1 @@
-export default ['Taiwan (R.O.C.)'];
+export default ['中國臺灣'];
```
Comment (ST-DDT) [inline src/locales/zh_TW/location/default_country.ts]: **Team decision**

We were informed that this value might be politically(?) sensitive.
As we aren't ourselves into this topic and the file is currently unused outside of the unconfirmed use via `fake`, we would like to keep the file as is (even if it is wrong), that way we are able to merge this PR faster.
We can then change this file in a separate PR later with focused community feedback.

For now, please revert this file. Sorry, for the trouble.

**Review by ST-DDT — CHANGES_REQUESTED** [review]

**Review by matthewmayer — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Comment by CoolPlayLin** [comment]
@Shinigami92 @xDivisionByZerox @import-brain Could you review this for me? Thanks a lot!

**Comment by matthewmayer** [comment]
It is quite normal that PRs in the Faker project can take weeks to get reviewed and merged even if they are 100% fine. 

You don't need to do anything, the maintainers will notify you if any changes are needed.

**Review by ejcheng — APPROVED** [review]

**Comment by ST-DDT** [comment]
> It is quite normal that PRs in the Faker project can take weeks to get reviewed and merged even if they are 100% fine.

I wouldnt limit this to Faker or OpenSource specifically. It is just a mater of available time and priorities.

**Comment by ST-DDT** [comment]
Thanks for your contribution. ❤️

**Pull request merged**